ansible/roles/os_zabbix/vars/template_clam_update.yml

```diff
@@ -0,0 +1,14 @@
+---
+g_template_ops_clam_update:
+  name: Template Ops Clam Update
+  zitems:
+  - key: clam.update.signatures.not.updating
+    applications:
+    - Ops Clam Update
+    value_type: int
+
+  ztriggers:
+  - name: "Clam signatures older than 2 weeks"
+    expression: "{Template Ops Clam Update:clam.update.signatures.not.updating.min(#3)}>0"
+    url: "https://github.com/openshift/ops-sop/blob/master/v3/alerts/check_clam_update.asciidoc"
```

Comment:
This link is not valid.

If it was, it should not be pushed into a file present in openshift-tools.

Reply:
Placeholder link. I wasn't going to write the SOP before it was known if the deadline for monitoring could be reached (or if we were even going forward with monitoring).